backend/src/main/java/com/redhat/cloud/notifications/processors/email/Email.java

```diff
@@ -1,70 +1,70 @@
-package com.redhat.cloud.notifications.processors.email.bop;
+package com.redhat.cloud.notifications.processors.email;
 
 import java.util.HashSet;
 import java.util.Set;
 
 /**
  * Input JSON format accepted by the BOP
  */
-public class Email {
+class Email {
```

Comment:
There are lots of access modifier changes and minor refactors in this PR which make the review longer/harder, especially in `EmailSubscriptionTypeProcessor`. Please don't put too many of these in a PR that contains a major change such as the daily email job. 😃

Reply:
Argh.. sorry. Got used to the pathfinder rule and clean up things the moment I see them.